src/Makefile

```diff
@@ -63,11 +63,13 @@ clean: $(patsubst %, %_clean, $(DIRS))
 $(patsubst %, %_clean, $(DIRS)):
 	$(MAKE) $(MAKEARGS) -C $(patsubst %_clean, %, $@) clean ; \
 
-# minisat 2 download, for your convenience
+# minisat2 and glucose download, for your convenience
+
+DOWNLOADER = lwp-download
 
```

Comment:
This enables the user to set an alternative download program without editing the Makefile.
E.g. to use wget on Windows under Cygwin, do
`make DOWNLOADER=wget minisat2-download`